quic/s2n-quic-tls/src/session.rs

```diff
@@ -155,6 +156,10 @@ impl tls::Session for Session {
                     context.on_tls_exporter_ready(self)?;
                     self.handshake_complete = true;
                 }
+                // TODO Add new s2n-tls new api, take and put in quic::connection
+                let ctx: Option<Box<dyn Any + Send + Sync>> =
```

Comment:
Does this need a `'static` as well?

Reply:
Any always has 'static https://doc.rust-lang.org/std/any/trait.Any.html